src/Middleware/HttpLogging/src/HttpLoggingExtensions.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.HttpLogging
+{
+    internal static class HttpLoggingExtensions
```

Comment:
We generally don't define logging extensions outside of the type that they're used. Why do this?

Reply:
I see this patter in dozen of places:
https://github.com/dotnet/aspnetcore/blob/main/src/Servers/Kestrel/Core/src/Internal/LoggerExtensions.cs
https://github.com/dotnet/aspnetcore/blob/main/src/Middleware/ResponseCaching/src/LoggerExtensions.cs
https://github.com/dotnet/aspnetcore/blob/main/src/Middleware/HostFiltering/src/LoggerExtensions.cs

Better question is why not?

Reply:
This pattern isn't good. We're moving away from it and moving it closer to where it used.

Reply:
Like this? https://github.com/dotnet/aspnetcore/wiki/Engineering-guidelines#logging-patterns

If so then the rest of the repo should be updated to match.

Reply:
It should be. Newer code uses the pattern. Paging @gfoidl 😄

Reply:
Will move these to the new pattern together with https://github.com/dotnet/aspnetcore/issues/31581